infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/LogReplicationSourceManager.java

```diff
@@ -76,13 +76,17 @@ public LogReplicationSourceManager(LogReplicationRuntimeParameters params,
                                        LogReplicationMetadataManager metadataManager,
                                        DataSender dataSender) {
 
+        // This runtime is used exclusively for the snapshot and log entry reader which do not require a cache
+        // as these are one time operations.
         this.runtime = CorfuRuntime.fromParameters(CorfuRuntimeParameters.builder()
                 .trustStore(params.getTrustStore())
                 .tsPasswordFile(params.getTsPasswordFile())
                 .keyStore(params.getKeyStore())
                 .ksPasswordFile(params.getKsPasswordFile())
                 .systemDownHandler(params.getSystemDownHandler())
-                .tlsEnabled(params.isTlsEnabled()).build());
+                .tlsEnabled(params.isTlsEnabled())
+                .cacheDisabled(true)
```

Comment:
was this cache used when reading the stream data to replicate?

Reply:
correct, and the read is done once and sent to the standby, any buffering for resent is held on LR's buffers so no need to get it from the cache.